sdcm/cluster_k8s/eks.py

```diff
@@ -334,6 +343,25 @@ def deploy(self):
         self.create_eks_cluster()
         LOGGER.info("Patch kubectl config")
         self.patch_kubectl_config()
+        LOGGER.info("Create storage class")
+        self.create_ebs_storge_class()
+        LOGGER.info("Create role for ebs-csi-driver")
+        self.create_role_for_ebs_csi_driver()
+
+    def create_ebs_storge_class(self):
+        if self.params.get('k8s_scylla_disk_class') in SUPPORTED_EBS_STORAGE_CLASSES:
```

Comment:
Need also to have `else` clause where we write warning message that we didn't create any storage class.
But, maybe, we want always to create a default one? Any, even if `gp2`?

Reply:
gp2 is the default one and we don't need to create it. this is not going to change according to EKS docs.

anyhow the title of this PR is a bit misleading, since what we want in this PR is to make the default gp2 working again,
gp3 is less of an issue (it's just copied it from the branch I was trying to work on gp3 initially

I'm still trying to make the installation of ebs-csi-driver to work as expected...

Reply:
RIght, the commit with the `gp3` mentioning is totally measling because it's part with the IAM stuff is needed in general, always and not exactly for the GP3.